src/Nethermind/Nethermind.Runner/Program.cs

```diff
@@ -135,7 +135,9 @@ private static void Run(string[] args)
                 IConfigProvider configProvider = BuildConfigProvider(app, loggerConfigSource, logLevelOverride, configsDirectory, configFile);
                 IInitConfig initConfig = configProvider.GetConfig<IInitConfig>();
                 IKeyStoreConfig keyStoreConfig = configProvider.GetConfig<IKeyStoreConfig>();
+                IPluginConfig pluginConfig = configProvider.GetConfig<IPluginConfig>();
 
+                pluginLoader.OrderPlugins(pluginConfig);
```

Comment:
pass config into PluginLoader constructor.
Order internally, no need for OrderPlugins method call.

Reply:
Can't because of circular dependencies